views/partials/social-media-buttons.ejs

```diff
@@ -0,0 +1,79 @@
+<div class="media-buttons-div">
```

Comment:
I would recommend to add some spacing on the buttons for mobile (or keep the relatively big) because it might be difficulty to accurately tap them on mobile.

Reply:
I was initially considering to make the buttons bigger, but unfortunately i could not figure out a way to increase the icons sizes. I kinda forgot about it entirely later on, will try doing something about it now that you pointed it out as well.